example/src/screens/Examples/AwareScrollView/index.tsx

```diff
@@ -37,16 +39,52 @@ export default function AwareScrollView({ navigation }: Props) {
     });
   }, []);
 
+  const snapToOffsets = [125, 225, 325, 425];
```

Comment:
Let's move this variable outside of the function (in this case it will not be re-created on every render and it'll improve performance)